packages/flutter/test/rendering/wrap_test.dart

```diff
@@ -12,17 +12,146 @@ void main() {
     expect(
       renderWrap.toStringDeep(minLevel: DiagnosticLevel.info),
       equalsIgnoringHashCodes(
-        'RenderWrap#00000 NEEDS-LAYOUT NEEDS-PAINT DETACHED\n'
-        '   parentData: MISSING\n'
-        '   constraints: MISSING\n'
-        '   size: MISSING\n'
-        '   direction: horizontal\n'
-        '   alignment: start\n'
-        '   spacing: 0.0\n'
-        '   runAlignment: start\n'
-        '   runSpacing: 0.0\n'
-        '   crossAxisAlignment: 0.0\n'
-      ),
+          'RenderWrap#00000 NEEDS-LAYOUT NEEDS-PAINT DETACHED\n'
```

Comment:
can you revert this?

Reply:
I can return it, but this is the usual autoformat for this code, and these changes will appear again if someone uses autoformat again. Can you explain in more detail what is wrong with this?

Reply:
The Flutter repo doesn't use the auto-formatter, see here: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#formatting. Since you didn't touch this code and since your change is not really improving readability it should be reverted.